scripts/privacyidea

```diff
@@ -988,7 +988,7 @@ def create_arguments():
                           "of length",
                           metavar="NUMBER",
                           type=int,
-                          default=0)
+                          default=6)
```

Comment:
Don't think to complicated. Are you aware of the @-parameter? You can run

    privacyidea @secrets.xt

where secrets.txt is a file that contains all your default parameters!